_unittest/test_35_MaxwellCircuit.py

```diff
@@ -114,3 +114,11 @@ def test_08_import_netlist(self):
         self.aedtapp.insert_design("SchematicImport")
         self.aedtapp.modeler.schematic.limits_mils = 5000
         assert self.aedtapp.create_schematic_from_netlist(self.netlist_file1)
+
+    def test_09_create_voltage_source(self):
+        id = self.aedtapp.modeler.schematic.create_vsin(name="voltage_source", value=3.14, angle=90)
```

Comment:
@DaveTwyman this method does not exist, you created create_v_sin and this doesn't return an id but rather a component.

Reply:
@gmalinve , 'create_vsin' was an earlier iteration of the method name before 'create_v_sin' was used. We also decided to move from id to component. I'll adjust all these changes then mark as resolved after I've pushed changes out.